windows/src/desktop/kmshell/xml/keyman.xsl

```diff
@@ -20,7 +20,13 @@
     <html xmlns="http://www.w3.org/1999/xhtml">
       <head>
         <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-        <title><xsl:value-of select="$locale/String[@Id='S_ConfigurationTitle']"/></title>
+		<style> 
+			* { font-family: <xsl:value-of select="($locale/String[@Id='SK_UIFontName'])[1]" />;
+			  } 
+		</style>
+        <title>
```

Comment:
Please restore the formatting from before (no whitespace inside the `<title></title>` element

Reply:
done